Students/Minseong/model/usersDAO.js

```diff
@@ -0,0 +1,9 @@
+import prisma from '../prisma/index'
+
+const findUsers = async() => {
+    console.log('Im in Dao')
+    const users = await prisma.$queryRaw(`SELECT * FROM users;`)
+    return users;
+}
```

Comment:
```suggestion

const findUsers = async() => {
    console.log('Im in Dao')
    const users = await prisma.$queryRaw(`SELECT id, name, email FROM users;`)
    return users;
}
```
실제 업무에서는 `*`은 사용하지 않습니다 :) 내가 사용할 데이터의 컬럼만 정확히 지칭해서 가져오셔야 합니담

Reply:
넵 필요한 부분만 지정하여 가져오게 수정 하였습니다!